design/sketch.webidl

```diff
@@ -687,11 +687,16 @@ interface GPUQueue {
 };
 
 // SwapChain / CanvasContext
-interface GPUCanvasContext {
+partial interface GPUCanvasContext {
+    // Calling createSwapChain a second time invalidates the previous one, an
+    // all of the textures it’s produced.
```

Comment:
Nit: "an all of the textures it's produced" --> "and all of the textures it's produced".

Reply:
Fixed. Thank you!